shared/test/test_hip_chat.rb

```diff
@@ -0,0 +1,80 @@
+require 'fakeweb'
+require 'minitest/autorun'
+require 'rack/test'
+require_relative '../../deployment'
+require_relative '../../lib/cdo/hip_chat'
+
+class HipchatTest < Minitest::Test
+
+  MAX_WAIT_SECONDS = 10
+
+  # Use a short retry backoff for this test to keep the test fast.
+  BACKOFF = 0.1
+
+  def setup
+    CDO.hip_chat_logging = true
+    CDO.log.level = 5  # Log only fatal exceptions to avoid test spew.
```

Comment:
Can you use the symbol equivalent here instead of a magic number?

http://guides.rubyonrails.org/debugging_rails_applications.html#log-levels